### DISCUSSION TITLE
我想使用delete_msg命令撤回消息 但是我只能撤回机器人（有管理员权限）发送的消息 无法撤回我发送的消息

### DISCUSSION THREAD

**Discussion by Ah-Q** [nonebot/discussions#450] [问答] [answered]
这是代码：
![image](https://github.com/nonebot/discussions/assets/80017370/b424e1a7-bd05-411d-9203-65cb5ccaaba0)
这是测试结果：
![image](https://github.com/nonebot/discussions/assets/80017370/c1273592-5fcf-4594-b10a-bd7654be964b)

请问是怎么回事呢

**Comment by yanyongyu** [comment, marked as answer]
请提供一下你的nb日志，看下调用接口发生了什么

**Reply by Ah-Q** [reply]
![image](https://github.com/nonebot/discussions/assets/80017370/6850c4ec-0fa4-4ccd-847a-c24fbfca5ffe)

**Reply by Ah-Q** [reply]
delete_msg调用了两次 send_msg调用了一次 成共发送了一条消息并且成功撤回了此条消息；但是没有撤回我发送的消息“测试”

**Reply by yanyongyu** [reply]
看上去api调用是成功的，你得看协议端日志了，估计是协议端的问题，你用的协议端是？

**Reply by Ah-Q** [reply]
### ### 这个是协议端日志：
![image](https://github.com/nonebot/discussions/assets/80017370/cd9de555-9680-4b8e-a51d-0ed0be15bd50)

**Reply by Ah-Q** [reply]
看上去好像也没什么问题

**Reply by yanyongyu** [reply]
额，你似乎是想撤回群主的消息。。。？
